vagrant/Vagrantfile

```diff
@@ -13,12 +13,16 @@
 HEAD_IP = "192.168.233.10"
 WORKER_IP = "192.168.233.11"
 
+Dir.mkdir("./data") if not Dir.exists?("./data")
+
 Vagrant.configure("2") do |config|
 
   config.vm.box = "geerlingguy/centos7"
   config.vm.box_version = "1.2.6"
 
   config.vm.synced_folder "..", "/usr/local/share/Kive"
+  config.vm.synced_folder "./data", "/data",
+    mount_options: ["dmode=777", "fmode=777"]
```

Comment:
Does that fmode make sense, or would 666 be better? I'm not familiar with the option, so I'm just guessing how it works. For Kive in particular, I can't think of any reason why we'd need executable files under the media root.

Reply:
As I understand it, the directory needs to be "executable" so that other users can enter and modify it.

Reply:
Maybe I'm misunderstanding what `fmode` and `dmode` are used for. I assumed that `fmode` would be applied to files and `dmode` to directories. Also, are they just replacing the existing file attributes or are they masking them? If they're used as a mask, then just add an explanatory comment and leave them as is.